### ISSUE TITLE
Done bucket doesn't work anymore

### ISSUE THREAD

**Issue by koalp** [go-vikunja/vikunja#280]
### Description

When creating a bucket in a kanban and marking it done, this bucket does not act as the done bucket anymore

Reproduce : 

- create a  project with two tasks.
- Add a "Done" bucket and set it as Done bucket
- Mark a task as done
- Move the other to the done bucket

You end up with a "done" task in the backlog bucket an "undone" task in the Done bucket.

### Vikunja Version

https://try.vikunja.io/  Version: v0.23.0-931-a721d9286b

### Can you reproduce the bug on the Vikunja demo site?

Yes

### Screenshots

![image](https://github.com/go-vikunja/vikunja/assets/49921473/c3e00cb5-4eb7-47d5-ad58-ee236851e804)

**Comment by kolaente** [comment]
Does that happen instantly or only after a refresh?

**Comment by koalp** [comment]
Yes, it is still reproductible on try (Version: v0.23.0-937-fe0e22a5cd)

[simplescreenrecorder-2024-07-04_18.09.19.webm](https://github.com/go-vikunja/vikunja/assets/49921473/1bfb24c3-5dcb-4c6e-a345-6be4a95a2437)

**Comment by koalp** [comment]
Closing as duplicate of https://kolaente.dev/vikunja/vikunja/issues/2518
